code/go/0chain.net/blobbercore/handler/object_operation_handler.go

```diff
@@ -1041,25 +1030,12 @@ func (fsh *StorageHandler) WriteFile(ctx context.Context, r *http.Request) (*blo
 
 	elapsedRef := time.Since(st)
 	st = time.Now()
+
 	connectionObj, err := allocation.GetAllocationChanges(ctx, connectionID, allocationID, clientID)
 	if err != nil {
 		return nil, common.NewError("meta_error", "Error reading metadata for connection")
 	}
 
```

Comment:
@Roshan-Mehta I don't see size update in connection object. Is it done somewhere for file upload?